### DISCUSSION TITLE
Continue training on existing model with new train data

### DISCUSSION THREAD

**Discussion by suheylkiris** [PaddlePaddle/PaddleOCR#9076] [Q&A] [answered]
Hello,

I am using some models which provides from PaddleOCR like en_PP-OCRv3_xx or ch_ppocr_server_v2.0_xx. Actually they are really useful but i have new some special cases and so i want to implement my new train data to this models. I try to train new model with Backbones but my local computer GPU is insufficient for train big datasets. Is it possible? If it is, should i use inference models? Could anybody share any guide about this. I have read almost all english documents but i was really confused and feeling like im lost.

**Comment by TuanBC** [comment, marked as answer]
If you have any change in terms of char_dict, language, or if the accuracy of the inference model is not good for you, then it maybe worth it to try finetune the pretrained model. With the original PPOCRv3 small model config (image size 48x320), it consumes about 19GB of memory on the GPU with batch_size=128, and the memory consumption will increase or decrease linearly with the batch_size. If there is no GPU available, you could try some online resources from Baidu or Google Colab.

**Reply by suheylkiris** [reply]
Thank you so much. I spent a lot of time with github documentations and E-Book, i figured out how can i train or fine-tune new model. But still i need some reading/explanation resources about two subject:
1) Where can i learn resource(GPU, CPU) consumption of models and how effects their parameters(batch_size, lr) etc... which like is exampled in your comment.
2) I did not understand teacher-student relations and algorithms. How it works actually and how can i configure config files? In what situations should I use it? How can i decide which algorithm is a teacher or student? Can each model be used as a teacher or student? etc etc....
